README.rdoc

```diff
@@ -27,8 +27,13 @@ Convert country names and codes to a standard.
 
  # Set the default
  NormalizeCountry.to = :alpha3
- NormalizeCountry.convert("Mexico")                 # "MEX"
- NormalizeCountry.convert("United Mexican States")  # "MEX"
+ NormalizeCountry.convert("Mexico")                # "MEX"
+ NormalizeCountry.convert("United Mexican States") # "MEX"
+
+ # Extend a list of default countries
+ NormalizeCountry.extend_countries(:alpha2 => "SU", :iso_name => "Soviet Union", :short => "USSR")
```

Comment:
I think that both variants `add_countries` and `extend_countries` are possible in this case.

We can also have two aliases:
* `add_countries` is more informatively when we want to append new countries.
* `extend_countries` is more informatively when we want to override rules in the existing list.